controller/cmd/proxy-injector/main.go

```diff
@@ -2,21 +2,30 @@ package proxyinjector
 
 import (
 	"context"
+	"flag"
+	"fmt"
 
 	"github.com/linkerd/linkerd2/controller/k8s"
 	injector "github.com/linkerd/linkerd2/controller/proxy-injector"
 	"github.com/linkerd/linkerd2/controller/webhook"
+	"github.com/linkerd/linkerd2/pkg/flags"
 )
 
 // Main executes the proxy-injector subcommand
 func Main(args []string) {
+	cmd := flag.NewFlagSet("proxy-injector", flag.ExitOnError)
+	metricsAddr := cmd.String("metrics-addr", fmt.Sprintf(":%d", 9995), "address to serve scrapable metrics on")
+	addr := cmd.String("addr", ":8443", "address to serve on")
+	kubeconfig := cmd.String("kubeconfig", "", "path to kubeconfig")
+	flags.ConfigureAndParse(cmd, args)
```

Comment:
The upcoming jaeger webhook will define a couple of extra flags here. We'll pass them to the handler function below (`Injector.Inject`)  in this case.